ParisBNF/abb/BNFabb146.xml

```diff
@@ -0,0 +1,344 @@
+                  <p>This file is licensed under the Creative Commons
+                                            Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno>BnF Éthiopien d'Abbadie 146</idno>   
+               </msIdentifier>
+       
+                  
+                  <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>
+                              
+                              <material key="parchment"/>                           
+                           </support>
+                           <extent>
+                              <measure unit="leaf"  quantity="119">119</measure>
+                              <measure unit="quire"/>
+                              <dimensions type="outer" unit="mm">
+                                 <height cert="low">120</height>
+                                 <width cert="low">125</width>                                 
+                              </dimensions>
+                           </extent>
+                           <foliation/>
+                           <collation/>                           
+                        </supportDesc>                        
+                        
+                     </objectDesc>
+                      <bindingDesc>
+                        <binding contemporary="non-Ethiopian" cert="medium">
+                           <decoNote xml:id="b1" type="Other">The catalogue of M. Chaîne indicates that the binding 
+                              is European. According to the earlier catalogue of A. d'Abbadie, 
+                             the manuscript was originally bound in leather, with the spine 
+                              protected by a cotton patch.</decoNote> 
+                        </binding>
+                     </bindingDesc>
+                     
+                  </physDesc>
+                  
+                  <history>
+                     <origin>
+                        <origDate notBefore="1700" notAfter="1799">18th century</origDate>.
+                        M. Chaîne dates the manuscript to 18th/19th century. The catalogues of M. Chaîne and C. Conti Rossini
+                        provide a number of details which are difficult to reconcile. The manuscript is composite, 
+                        but the exact number of units and scribal hands involved can 
+                        hardly be established, on the basis of the available descritions. 
```

Comment:
These comments are better to be placed in handDesc, contents or collation.

Reply:
I removed the statement on the scribal hands, units is ok, I assume.

Reply:
Change to "description". I have seen this just now.